lib/flowViewport/API/perViewportSceneIndicesData/fvpDataProducerSceneIndexDataBase.h

```diff
@@ -113,6 +113,8 @@ TF_DECLARE_WEAK_AND_REF_PTRS(DataProducerSceneIndexDataBase);//Be able to use Re
     std::string                         _rendererNames;
     /// Is the DCC node so a MObject* DAG node for Maya
     void*                               _dccNode;
+    /// Is the DCC node hashcode (we use it to remove the node from a registry when it has been deleted)
+    unsigned long                       _dccNodeHashCode = 0;
```

Comment:
Is the hash code implemented with a pointer?  The C++ standard says that this must be at least 32 bits, but unsigned long long would be the proper type for 64 bits:
https://en.cppreference.com/w/cpp/language/types
Either that or std::uintptr_t.